.travis.yml

```diff
@@ -26,10 +28,15 @@ script: >
     chmod +x check_headers.py
     python3 ./check_headers.py -f "$(git diff --name-only --diff-filter=d $TRAVIS_COMMIT_RANGE)"
   fi &&
-  if [ x${TEST} = xunit_test ]; then
+
+  if [ x${TEST} = xbuild ]; then
     make build
```

Comment:
Should check the result of command execution here?

Reply:
Pipeline will check the execution status
Done. Your build exited with 0 or
Done. Your build exited with 1

Reply:
Could you check this PR https://github.com/vmware/singleton/pull/1077?
The CI is hanging.